### ISSUE TITLE
Dataset curation does not contain Jiang24 and Frangieh21

### ISSUE THREAD

**Issue by PancakeZoy** [altoslabs/perturbench#6]
I didn't find the curation script for the following two datasets: Jiang24 and Frangieh21.

I also did no find the notebook mentioned in your manuscript: "notebooks/neurips2024/build_data_scaling_splits.ipynb"

Can you please let me know if my observation is correct? Thank you

**Comment by yanwu2014** [comment]
Yes apologies for that we're still updating the repo, hoping to have these updates pushed early next year!

**Comment by yanwu2014** [comment]
@PancakeZoy curation notebooks should be uploaded now, lmk if you have any questions or run into any issues! Closing this issue for now but feel free to reopen if necessary
